master/internal/db/postgres_tasks_intg_test.go

```diff
@@ -733,6 +733,23 @@ func TestTaskLogsFlow(t *testing.T) {
 	require.NoError(t, err)
 	require.Len(t, logs, 2)
 
+	// Filter by search text.
+	logs, _, err = db.TaskLogs(t1In.TaskID, 5, []api.Filter{{
+		Field:     "log",
+		Operation: api.FilterOperationStringContainment,
+		Values:    []string{"^th.s"},
+	}}, apiv1.OrderBy_ORDER_BY_UNSPECIFIED, nil)
+	require.NoError(t, err)
+	require.Empty(t, logs)
```

Comment:
I like that you added a `FilterOperationStringContainment` test, but can you add one that shows that the feature works?  This test shows that it doesn't work with regexes, which is not a particularly useful test I think.